momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -922,13 +925,133 @@ public CompletableFuture<CacheSortedSetGetRankResponse> sortedSetGetRank(
    * @param element - The element whose rank we are retrieving.
    * @param order - The order to read through the scores of the set. Affects the rank. Defaults to
    *     ascending, i.e. the rank of the element with the lowest score will be 0.
-   * @return Future containing the result of the fetch operation.
+   * @return Future containing the result of the get rank operation.
    */
   public CompletableFuture<CacheSortedSetGetRankResponse> sortedSetGetRank(
       String cacheName, String sortedSetName, byte[] element, @Nullable SortOrder order) {
     return scsDataClient.sortedSetGetRank(cacheName, sortedSetName, element, order);
   }
 
+  /**
+   * Look up the score of an element in a sorted set.
+   *
+   * @param cacheName - The cache containing the sorted set.
+   * @param sortedSetName - The sorted set to fetch from.
```

Comment:
nit that can be dealt with later - lots of the javadocs say "fetch"

Reply:
I'm sure there are a few copy/paste errors. There are also 77 javadoc warnings on build that have been annoying me. I'll do a pass over the docs after retries go in.

Reply:
👍